.babelrc

```diff
@@ -1,7 +1,8 @@
 {
 	"presets": [
 		"es2015",
-		"stage-2"
+		"stage-2",
+		"flow"
```

Comment:
We only don't use `babel-preset-react` currently because it [bundles the Flow preset](https://www.npmjs.com/package/babel-preset-react), meaning if we're opting in to Flow now, we could simply use the React preset and eliminate a number of the individual plugins.